src/chocolatey/infrastructure.app/commands/ChocolateyExportCommand.cs

```diff
@@ -161,6 +180,49 @@ public void run(ChocolateyConfiguration configuration)
+                                    _nugetService.set_package_config_for_upgrade(configuration, pkgInfo);
+
+                                    //Mirrors the arguments captured in ChocolateyPackageService.capture_arguments()
+
+                                    if (configuration.Prerelease) xw.WriteAttributeString("prerelease", "true");
+                                    if (configuration.IgnoreDependencies) xw.WriteAttributeString("ignoreDependencies", "true");
+                                    if (configuration.ForceX86) xw.WriteAttributeString("forceX86", "true");
+
+                                    if (!string.IsNullOrWhiteSpace(configuration.InstallArguments)) xw.WriteAttributeString("installArguments", configuration.InstallArguments);
+                                    if (configuration.OverrideArguments) xw.WriteAttributeString("overrideArguments", "true");
+                                    if (configuration.ApplyInstallArgumentsToDependencies) xw.WriteAttributeString("applyInstallArgumentsToDependencies", "true");
+
+                                    if (!string.IsNullOrWhiteSpace(configuration.PackageParameters)) xw.WriteAttributeString("packageParameters", configuration.PackageParameters);
+                                    if (configuration.ApplyPackageParametersToDependencies) xw.WriteAttributeString("applyPackageParametersToDependencies", "true");
+
+                                    if (configuration.AllowDowngrade) xw.WriteAttributeString("allowDowngrade", "true");
+                                    if (configuration.AllowMultipleVersions) xw.WriteAttributeString("allowMultipleVersions", "true");
+
+                                    if (!string.IsNullOrWhiteSpace(configuration.SourceCommand.Username)) xw.WriteAttributeString("user", configuration.SourceCommand.Username);
+                                    if (!string.IsNullOrWhiteSpace(configuration.SourceCommand.Password)) xw.WriteAttributeString("password", configuration.SourceCommand.Password);
+                                    if (!string.IsNullOrWhiteSpace(configuration.SourceCommand.Certificate)) xw.WriteAttributeString("cert", configuration.SourceCommand.Certificate);
+                                    if (!string.IsNullOrWhiteSpace(configuration.SourceCommand.CertificatePassword)) xw.WriteAttributeString("certPassword", configuration.SourceCommand.CertificatePassword);
+
+                                    //Arguments from the global options set
+                                    if (configuration.CommandExecutionTimeoutSeconds != ApplicationParameters.DefaultWaitForExitInSeconds)
+                                    {
+                                        xw.WriteAttributeString("timeout",configuration.CommandExecutionTimeoutSeconds.to_string());
+                                    }
+
+                                    //TODO, should this be exported? It seems rather system specific

```

Comment:
Can we add a space here after the `//`?

Reply:
Regarding the actual question on this one...

This is a good question! When actually using the remembered arguments during an upgrade procedure, using the same cachelocation used during the installation makes sense. However, if exporting the configuration and running  on a new machine, that could indeed be different.  I would be inclined to leave this commented out, and replace the TODO comment with another comment that says that we at least thought about this, and decided to not export it, and left it here commented out to ensure that it wasn't forgotten about in future.

Reply:
Done